src/Utilities/CssConcatenator.php

```diff
@@ -66,25 +72,21 @@ public function append(array $selectors, string $declarationsBlock, string $medi
         $selectorsAsKeys = \array_flip($selectors);
 
         $mediaRule = $this->getOrCreateMediaRuleToAppendTo($media);
-        /** @var array<int, \stdClass> $ruleBlocks */
         $ruleBlocks = $mediaRule->ruleBlocks;
         $lastRuleBlock = \end($ruleBlocks);
 
-        $hasSameDeclarationsAsLastRule = $lastRuleBlock instanceof \stdClass
+        $hasSameDeclarationsAsLastRule = \is_object($lastRuleBlock)
             && $declarationsBlock === $lastRuleBlock->declarationsBlock;
         if ($hasSameDeclarationsAsLastRule) {
-            /** @var array<string, array-key> $lastRuleBlock->selectorsAsKeys */
             $lastRuleBlock->selectorsAsKeys += $selectorsAsKeys;
         } else {
-            /** @var array<string, array-key> $lastRuleBlockSelectors */
-            $lastRuleBlockSelectors = $lastRuleBlock instanceof \stdClass ? $lastRuleBlock->selectorsAsKeys : [];
-            $hasSameSelectorsAsLastRule = $lastRuleBlock instanceof \stdClass
+            $lastRuleBlockSelectors = \is_object($lastRuleBlock) ? $lastRuleBlock->selectorsAsKeys : [];
+            $hasSameSelectorsAsLastRule = \is_object($lastRuleBlock)
```

Comment:
Please keep the `instanceof` checks here as those are more specific than `is_object`.

Reply:
Again, Psalm complained about `\stdClass` not being the same as `object`.  But if `\stdClass` can be used in the type annotation, these can be reverted.

Reply:
Unfortunately, Psalm doesn't recognize `\stdClass{property: value-type}`, but if these use `instanceof \stdClass` it seems to forget all of the type information provided in the DocBlock and go on to report errors like

> ERROR: MixedOperand - src/Utilities/CssConcatenator.php:81:13 - Left operand cannot be mixed (see https://psalm.dev/059)

whereever the properties are accessed in a code path that would be executed in the case that `instanceof \stdClass` is true.

However, I'm seeing many more Psalm errors than you are (I have no idea why, I am using the same Psalm version), so I'll see what happens on the server...